kernel/static/index.html

```diff
@@ -497,6 +497,7 @@
       document.getElementById('eth-login-confirm-button').disabled = !document.getElementById('agree-check').checked;
     }
   </script>
+  <script>window.VERSION = 'EXPLORER_VERSION';</script>
```

Comment:
I think we can't use the word `EXPLORER_VERSION` here because it is replace with the commit hash. I'm not sure, I'll check it